3p/integration.js

```diff
@@ -208,6 +208,7 @@ const defaultAllowedTypesInCustomFrame = [
 export const waitForRenderStart = [
   'doubleclick',
   'fakead3p',
+  'inmobi',
```

Comment:
did you sync to the latest code? we recently moved this array to a different file.